sandbox/urls.py

```diff
@@ -26,6 +27,16 @@
 
 
 urlpatterns = [
+    # Add sitemap.xml URL to the robots.txt so we don't need to register sitemap.xml from each 
+    # crawler administration panel
+    path(
+        r"robots.txt",
+        lambda request: HttpResponse(
```

Comment:
Can we make it a real template for clarity?
Also note that this `urls.py` file is just for sandbox purposes so it will just serve as an example but each site will have its own `urls.py` file it its project.

Reply:
Ok, no problem. Done.